frontend/src/_layouts/base.html

```diff
@@ -1,118 +1,161 @@
+      <link rel="icon" sizes="32x32" type="image/png" href="~/src/images/favicon-32x32.png">
+      <link rel="icon" sizes="16x16" type="image/png" href="~/src/images/favicon-16x16.png">
+      <link rel="stylesheet" href="~/src/app.css">
+      <!-- <link href="output.css" rel="stylesheet">-->
+  <!--<script src="https://code.iconify.design/iconify-icon/1.0.2/iconify-icon.min.js"></script>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.4/flowbite.min.css" rel="stylesheet" />
+  <script src="node_modules/flowbite/dist/flowbite.min.js"></script>-->
+  <title>Srila BV Narayana Gosvami Maharaja’s Official Archive</title>
+  {% render 'partials/gtm.html', tag-id: gtm-tag-id -%}
+</head>
+<body>
+  {% render 'partials/gtm-noscript.html', tag-id: gtm-tag-id -%}
+  <!--Top section Join us now-->
+  {% if bannerShow != false %}
+  <button data-drawer-target="sidebar-multi-level-sidebar" data-drawer-toggle="sidebar-multi-level-sidebar"
+  aria-controls="sidebar-multi-level-sidebar" type="button"
+  class="inline-flex items-center p-3 border-b border-b-white text-sm text-gray-900 !bg-pink-100 !w-full font-extrabold md:hidden
+   hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-600 dark:hover:bg-gray-700 dark:focus:ring-gray-600">
+  <span class="sr-only">Open sidebar</span>
+  <svg class="w-6 h-6" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20"
+    xmlns="http://www.w3.org/2000/svg">
+    <path clip-rule="evenodd" fill-rule="evenodd"
+      d="M2 4.75A.75.75 0 012.75 4h14.5a.75.75 0 010 1.5H2.75A.75.75 0 012 4.75zm0 10.5a.75.75 0 01.75-.75h7.5a.75.75 0 010 1.5h-7.5a.75.75 0 01-.75-.75zM2 10a.75.75 0 01.75-.75h14.5a.75.75 0 010 1.5H2.75A.75.75 0 012 10z">
+    </path>
+  </svg>
+</button>
+{% endif %}
+  <aside id="sidebar-multi-level-sidebar"
+        class="fixed top-0 left-0 z-40 w-64 h-screen transition-transform -translate-x-full sm:translate-x-0 md:w-auto md:relative md:h-auto md:bg-none md:top-auto "
+        aria-label="Sidebar">
+        <div class=" h-full overflow-y-auto bg-white ">
+  <section class="!w-full  md:!border-b !border-t !border-gray-300 md:!bg-emerald-700/10 mx-auto h-15 md:!h-12 items-center justify-center
+  space-x-2 md:flex  px-4 md:!px-0 md:!text-left pt-4 md:!pt-0 absolute left-0 bottom-4 md:!left-auto md:!bottom-auto md:!relative ">
+    <div class="font-normal md:!text-sm text-xs"><span class="md:!text-sm text-xs font-bold">JOIN US&nbsp;</span> for Daily Harikatha & Short
+      Videos in Hindi & English @ Gaudiya Rasamrita</div>
+    <div class="flex md:justify-center items-center md:!space-x-2 text-gray-700 mt-1 !space-x-1 !justify-start ">
+      <!--Using the below icons from https://iconify.design which is open source and all are easily customizable.-->
+      <a href="https://chat.whatsapp.com/DVLt0xX4yab3acLQRUBoBv" target="_blank">
+        <iconify-icon icon="ic:twotone-whatsapp" class="text-xl -ml-2 md:!-ml-0"></iconify-icon>
```

Comment:
Instead of custom elements, let's utilize `iconify`'s integration with Tailwind: https://docs.iconify.design/usage/css/tailwind/

Also, there are some icons that we include as `svg` elements, for them we also need to find an iconify replacement.